src/sql/parts/dashboard/dashboardInput.ts

```diff
@@ -90,6 +90,9 @@ export class DashboardInput extends EditorInput {
 			// Only add DB name if this is a non-default, non-master connection
 			name = name + ':' + this.connectionProfile.databaseName;
 		}
+		if (this.connectionProfile.connectionName !== null &&  this.connectionProfile.connectionName !== undefined){
```

Comment:
could streamline this using "truthy" test.  Also would be better to pull the connectionName test up a level and then keep a single 'return' statement with name set to either the connection name or the dynamically built name.